### DISCUSSION TITLE
How to DL One Ep At A Time + Rename The Mp3s?

### DISCUSSION THREAD

**Discussion by GitAlexToTheGreek** [gpodder/gpodder#1598] [Q&A]
Hi there,

1) Using a fast pc with a fast isp connection, but attempting to download more than one episode at a time grinds the whole system to a halt...

⏩ How to adjust the settings to dl one at a time? 

2) Mp3s are downloaded/stored as "d2c7a93548217bc1fd46d646a3f0e0a0.mp3" for example...

⏩ Possible to adjust the settings to have them saved using the extensive title/date/artist, etc details from the published episodes?

3) It's unclear, from either the GP preferences, podcast settings or config, how to change the default download folder location?
(I see how to *open* the designated location, but not change it.)

⏩ How to do this?

Thanks in advance!

**Comment by auouymous** [comment]
Answered on IRC.
